### PR TITLE
fix issue setting saved logits

### PR DESCRIPTION
**Pull request by jameswex**
This change is [Reviewable](https://reviewable.io/reviews/pair-code/deeplearnjs/796)

### CODE REVIEW

**Comment by nsthorat** [comment]
:lgtm_strong:

---

Review status: 0 of 2 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pair-code/deeplearnjs/796#-:-L6WvwrN3WFya-02X2dK:b-jbsf8z)*
